tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -49,9 +49,18 @@ PointerVariableConstraint::PointerVariableConstraint(const Type *_Ty,
   // If, after boiling off the pointer-ness from this type, we hit a 
   // function, then create a base-level FVConstraint that we carry 
   // around too.
-  if (Ty->isFunctionType()) {
+  if (Ty->isFunctionType()) 
+    // For the moment, if a type is typedefed, we don't need to print 
```

Comment:
I think this comment is still a little unclear.  How about:  "C function-pointer type declarator syntax embeds the variable name within the function-like syntax.  For example:
void (*fname)(int, int) = ...;
If a typedef'ed type name is used, the name can be omitted because it is not embedded like that.  Instead, it has the form
tn fname = ...,
where tn is the typedef'ed type name.